content/docs/add-react-to-a-website.md

```diff
@@ -1,35 +1,34 @@
 ---
 id: add-react-to-a-website
-title: Add React to a Website
+title: React को Website से जोड़े
```

Comment:
`से जोड़े` => `ऐड करें`

Reply:
ok I'll change this

Reply:
`करे` => `करें`

minor spelling mistake -> missing dot